packages/sdk/src/Config.ts

```diff
@@ -9,22 +9,33 @@ import validate from './generated/validateConfig'
 import { GapFillStrategy } from './subscribe/ordering/GapFiller'
 import { config as CHAIN_CONFIG } from '@streamr/config'
 import { CONFIG_TEST } from './ConfigTest'
+import { Identity } from './identity/Identity'
+import { KeyType } from './identity/identityConfig'
 
-export interface ProviderAuthConfig {
-    /**
-     * The {@link https://docs.ethers.org/v6/api/providers/#Eip1193Provider Eip1193Provider} type is from the `ethers` library.
-     */
+/**
+ * For passing in an Ethereum provider (= wallet) for signing
+ */
```

Comment:
Could keep the link to the `ethers.js` documentation so that we don't have undocumented types in the API.

Typically the instance passed here is a MetaMask's global `window.ethereum` object.

```suggestion
/**
 * For passing in an Ethereum provider (e.g. MetaMask) for signing
 *
 * The {@link https://docs.ethers.org/v6/api/providers/#Eip1193Provider Eip1193Provider} type is from the `ethers` library.
 */
```

Reply:
My issue here was that the argument has nothing to do with `ethers` - we only borrow the type definition from ethers. Any type definition that complies with the [spec](https://eips.ethereum.org/EIPS/eip-1193) is equally good. But I'll update the comment to clarify what is expected while keeping the ethers reference.